src/Microsoft.DotNet.Build.Tasks/PackageFiles/coverage/xunit.console.netcore.runtimeconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+	"runtimeOptions": {
+		"framework": {
+			"name": "Microsoft.NETCore.App",
+			"version": "10.10.10"
```

Comment:
Is there are reason you chose 10.10.10? I don't have any particular concerns just curious about the choice.

Reply:
I just needed it to be bigger than 2.* something so the runtime didn't complain, since 9.9.9 already existed I took the next one.